test/config/config.yaml

```diff
@@ -31,6 +31,27 @@ peers:
       p2p: 18333
       zmq: 28332
 
+broadcasting:
+  mode: unicast
```

Comment:
Maybe we should remove the `peers` config above the `broadcasting` config?

Reply:
Yes, `peers` can be removed